### DISCUSSION TITLE
How do I make wallpapers?

### DISCUSSION THREAD

**Discussion by STKKfred** [rocksdanister/lively#294] [Q&A] [answered]
I've come here to ask if there is software or anything to make wallpapers for lively. I have had a few ideas for some wallpapers and I can't find any info on making live wallpapers. Has anyone got any answers?

**Comment by rocksdanister** [comment, marked as answer]
https://www.reddit.com/r/LivelyWallpaper/comments/l91lp1/two_dumb_questions_for_yall
